grpc/src/test/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunctionBuilderTest.java

```diff
@@ -140,17 +144,19 @@ void mapStatusAndMetadata() {
                         })
                         .build();
 
-        final Status status = Status.UNKNOWN.withCause(new B1Exception());
-
+        final B1Exception cause = new B1Exception();
         final Metadata metadata1 = new Metadata();
-        final Status newStatus1 = GrpcStatus.fromExceptionHandler(exceptionHandler, ctx, status, metadata1);
+        final Status newStatus1 = exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault())
+                                                  .apply(ctx, cause, metadata1);
```

Comment:
```suggestion
        final Status newStatus1 = exceptionHandler.apply(ctx, cause, metadata1);
```

Reply:
Applied feedback, Thanks.
I've checked that test is working fine after change.